src/runtime/utils.ts

```diff
@@ -11,3 +13,43 @@ export const IS_BROWSER = typeof document !== "undefined";
 export function asset(path: string) {
   return `${INTERNAL_PREFIX}${STATIC_PREFIX}/${__FRSH_BUILD_ID}${path}`;
 }
+
+// apply the asset function a srcset
+export function assetSrcSet(srcset: string) {
+  const srcsetParts = srcset.split(",");
+  return srcsetParts.map((part) => {
+    const sections = part.trim().split(" ");
+    sections[0] = asset(sections[0]);
+    return sections.join(" ");
+  }).join(", ");
+}
+
+export function assetHashingHook(vnode: VNode) {
+  if (vnode.type === "img" || vnode.type === "source") {
+    const props = (vnode.props as HTMLImageElement);
+    // deno-lint-ignore no-explicit-any
+    if ((props as any)["data-no-auto-hashing"]) return;
```

Comment:
You should be able to get rid of the explicit any with this:
 
```
    if (props.dataset.noAutoHashing) return;
```
As per:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

Reply:
I've not managed to test this at runtime though, but it does stop the typescript error.

Reply:
That probably doesn't work, considering this is not a "real" `HTMLImageElement` on the server.